docs/explanations/how_to_interpret_the_security_status_command.rst

```diff
@@ -208,63 +208,48 @@ source):
 Machine-readable output
 =======================
 
-If you run the ``pro security-status --format yaml`` command on your machine,
-you should expect to see an output that follows this structure:
-
-.. code-block:: text
-
-    _schema_version: '0.1'
-    summary:
-      num_esm_apps_packages: 0
-      num_esm_apps_updates: 0
-      num_esm_infra_packages: 1
-      num_esm_infra_updates: 1
-      num_main_packages: 70
-      num_multiverse_packages: 10
-      num_restricted_packages: 10
-      num_third_party_packages: 0
-      num_universe_packages: 9
-      num_installed_packages: 100
-      num_standard_security_updates: 0
-      ua:
-        attached: true
-        enabled_services:
-        - esm-apps
-        - esm-infra
-        entitled_services:
-        - esm-apps
-        - esm-infra
-    packages:
-    - origin: esm.ubuntu.com
-      package: zlib1g
-      service_name: esm-infra
-      status: upgrade_available
-      version: 1:1.2.8.dfsg-2ubuntu4.3+esm1
-      download_size: 123456
-    livepatch:
-      fixed_cves:
-        - Name: cve-2013-1798
-          Patched: true
-
-Let's understand what each key means in the output of the
-``pro security-status --format yaml`` command:
-
-``summary``
------------
-
-This provides a summary of the system related to Ubuntu Pro and the different
-package sources in the system:
+If you need a machine readable version of ``pro security-status``, you can use
+these API endpoint to achieve that:
+
+* :ref:`u.pro.packages.summary.v1 <references/api:u.pro.packages.summary.v1>`
+* :ref:`u.pro.packages.updates.v1 <references/api:u.pro.packages.updates.v1>`
+* :ref:`u.pro.status.is_attached.v1 <references/api:u.pro.status.is_attached.v1>`
+* :ref:`u.pro.status.enabled_services.v1 <references/api:u.pro.status.enabled_services.v1>`
+* :ref:`u.pro.security.status.livepatch_cves.v1 <references/api:u.pro.security.status.livepatch_cves.v1>`
+
```

Comment:
Sorry, but I think we should delete everything after this line. If any of this detail is useful it can be added to the api reference IMO

Reply:
I think it makes sense for that to be the eventual goal, but can we have this as an interim step while we queue up the follow-up to fold this content into the API docs? Or would you prefer we do it now?

Reply:
I'm okay with this as an interim step!